stable/mssql-linux/values.yaml

```diff
@@ -16,13 +16,16 @@ persistence:
   # existingDataClaim:
   # existingTransactionLogClaim:
   # existingBackupClaim:
+  # existingMasterClaim:
   # storageClass: "-"
   dataAccessMode: ReadWriteOnce
   dataSize: 1Gi
   transactionLogAccessMode: ReadWriteOnce
   transactionLogSize: 1Gi
   backupAccessMode: ReadWriteOnce
   backupSize: 1Gi
+  masterAccessMode: ReadWriteOnce
```

Comment:
Please follow the best practises regarding persistence. I know you're just following the rest of the code here, but this would be a good time to change the code to adhere to best practises: https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md#persistence

Reply:
updated the version and following the best practice.

Reply:
Updated based on your suggestions